src/OpenSSL/SSL.py

```diff
@@ -1486,7 +1488,13 @@ def sendall(self, buf, flags=0):
         data = _ffi.new("char[]", buf)
 
         while left_to_send:
-            result = _lib.SSL_write(self._ssl, data + total_sent, left_to_send)
```

Comment:
I believe this change can be rolled back, since we bail out if left_to_send is to big, no?